### PR TITLE
Addition of PySimpleGUI to list of packages

### PR DESCRIPTION
**Pull request by PySimpleGUI**

### CODE REVIEW

File: packages/PySimpleGUI.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+repo: matplotlib/cmocean
+section: gui applications
+description: Create custom GUIs quickly & easily on top of tkinter, Qt, WxPython or Remi
+site: www.PySimpleGUI.com
```
Comment (jklymak) [inline packages/PySimpleGUI.yml:4]: Perhaps we should fix this, but `site` needs a protocol (http or https)...

File: packages/PySimpleGUI.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+repo: matplotlib/cmocean
```
Comment (jklymak) [inline packages/PySimpleGUI.yml:1]: We also need your GitHub repo...

**Comment by PySimpleGUI** [comment]
Wow.. .I'm really sorry... I was working from the example page and saw a match and moved on.... not paying close attention clearly.  🤦‍♂️

**Comment by jklymak** [comment]
No problem!  Thanks!

**Pull request merged**